### ISSUE TITLE
[a11y] Underline Links in Text

### ISSUE THREAD

**Issue by chmst** [joomla/Manual#341]
Links in text should be underlined, not only have a colour. 
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

![Image](https://github.com/user-attachments/assets/0bf59a16-1128-4e72-9e8d-8e7208ef102b)

Is it possible to do this automatically?

**Comment by HLeithner** [comment]
Sure it's possible to adapt the style with the custom.css file.

https://github.com/joomla/Manual/blob/main/src/css/custom.css

if you like and have time, please create a PR
